cmd/libsnap-confine-private/snap-test.c

```diff
@@ -118,13 +118,13 @@ static void test_sc_is_hook_security_tag(void)
 	g_assert_true(sc_is_hook_security_tag
 		      ("snap.foo_instance.hook.bar-baz"));
 	g_assert_true(sc_is_hook_security_tag("snap.foo_bar.hook.bar-baz"));
+	g_assert_true(sc_is_hook_security_tag("snap.name.app.hook.f00"));
```

Comment:
is this form really correct? I would have assumed that a correct instance would have been

```
	g_assert_true(sc_is_hook_security_tag("snap.foo.hook.bar-baz0"));
```

I don't see how the specified regex can match the `name.app.hook` ? there seems to be too many `.` to me

see https://regex101.com/r/hRgXlI/2

Reply:
indeed the unit tests are unhappy: 

```
# ERROR:libsnap-confine-private/snap-test.c:121:test_sc_is_hook_security_tag: 'sc_is_hook_security_tag("snap.name.app.hook.f00")' should be TRUE
```